<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>VidSummarizer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="setInputBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Set Input Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setOutputBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Set Output Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="summarizeBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Summarize</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setCompressionBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Set Compression Ratio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setSummarizationBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>390</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Set Summarization Method</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QTextEdit" name="summarizeText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>751</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar:vertical {
    background: rgb(245, 245, 245);
    width: 14px;
	border-radius: 7px;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(209, 209, 209);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(186, 186, 186);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(120, 120, 120);
}

QScrollBar::sub-line:vertical {
	background: none;
}

QScrollBar::add-line:vertical {
	background: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}

QScrollBar:horizontal {
	border: 1px solid rgb(245, 245, 245);
    background: rgb(245, 245, 245);
    height: 17px;
	border-radius: 7px;
 }

QScrollBar::handle:horizontal {	
	border: 1px solid rgb(209, 209, 209);
	background-color: rgb(209, 209, 209);
	height:14px;
	min-width: 30px;
	border-radius: 7px;
}
QScrollBar::handle:horizontal:hover{	
	background-color: rgb(186, 186, 186);
}
QScrollBar::handle:horizontal:pressed {	
	background-color: rgb(120, 120, 120);
}

QScrollBar::sub-line:horizontal {
	background: none;
}

QScrollBar::add-line:horizontal {
	background: none;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
	background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}

QTextEdit{
	border: 1px solid black;
	border-radius:50%;
	padding:15;
}

</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Summarized Text</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="videoText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>751</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar:vertical {
    background: rgb(245, 245, 245);
    width: 14px;
	border-radius: 7px;
 }

QScrollBar::handle:vertical {	
	background-color: rgb(209, 209, 209);
	min-height: 30px;
	border-radius: 7px;
}
QScrollBar::handle:vertical:hover{	
	background-color: rgb(186, 186, 186);
}
QScrollBar::handle:vertical:pressed {	
	background-color: rgb(120, 120, 120);
}

QScrollBar::sub-line:vertical {
	background: none;
}

QScrollBar::add-line:vertical {
	background: none;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}

QScrollBar:horizontal {
	border: 1px solid rgb(245, 245, 245);
    background: rgb(245, 245, 245);
    height: 17px;
	border-radius: 7px;
 }

QScrollBar::handle:horizontal {	
	border: 1px solid rgb(209, 209, 209);
	background-color: rgb(209, 209, 209);
	height:14px;
	min-width: 30px;
	border-radius: 7px;
}
QScrollBar::handle:horizontal:hover{	
	background-color: rgb(186, 186, 186);
}
QScrollBar::handle:horizontal:pressed {	
	background-color: rgb(120, 120, 120);
}

QScrollBar::sub-line:horizontal {
	background: none;
}

QScrollBar::add-line:horizontal {
	background: none;
}

QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
	background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}

QTextEdit{
	border: 1px solid black;
	border-radius:50%;
	padding:15;
}

</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Video Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>510</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:25%;border:1px solid black</string>
      </property>
      <property name="text">
       <string>Play Summarized Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>105</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Transcription</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>306</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Summarization</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
